Traceback (most recent call last):
  File "snippet.py", line 56, in <module>
    start_compressing()
  File "snippet.py", line 16, in start_compressing
    bucket = conn.get_bucket(BUCKET_NAME)
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 509, in get_bucket
    return self.head_bucket(bucket_name, headers=headers)
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 528, in head_bucket
    response = self.make_request('HEAD', bucket_name, headers=headers)
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 661, in make_request
    host = self.calling_format.build_host(self.server_name(), bucket)
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 88, in build_host
    return self.get_bucket_server(server, bucket)
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 67, in wrapper
    if len(args) == 3 and check_lowercase_bucketname(args[2]):
  File "/usr/local/lib/python2.7/site-packages/boto/s3/connection.py", line 59, in check_lowercase_bucketname
    raise BotoClientError("Bucket names cannot contain upper-case " \
boto.exception.BotoClientError: BotoClientError: Bucket names cannot contain upper-case characters when using either the sub-domain or virtual hosting calling format.
